code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -143,34 +149,39 @@
 		return
 
 	var/heal_prob = max(0, 80 - getCloneLoss())
-	var/mut_prob = min(80, getCloneLoss()+10)
-	if (amount > 0)
-		if (prob(mut_prob))
-			var/list/obj/item/organ/external/candidates = list()
-			for (var/obj/item/organ/external/O in organs)
-				if(O.status & ORGAN_ROBOT) continue
+	var/mut_prob = min(80, getCloneLoss() + 10)
+	if(amount > 0) //cloneloss is being added
+		if(prob(mut_prob))
+			var/list/obj/item/organ/external/candidates = list() //TYPECASTED LISTS ARE NOT A FUCKING THING WHAT THE FUCK

```

Comment:
this is only for decoration, p.sure.
a way to say "this is a list of limbs"
granted you can only tell here at the definition, where as a new name like ```candidate_limbs``` lets you figure it out wherever you see the var.

tl;dr they do, but it's for decoration.